src/convert/transformers/staticResourceMetadataTransformer.ts

```diff
@@ -39,6 +39,9 @@ export class StaticResourceMetadataTransformer extends BaseMetadataTransformer {
   public async toMetadataFormat(component: SourceComponent): Promise<WriteInfo[]> {
     const { content, type, xml } = component;
 
+    // archiver/zip.finalize is async.  If you await it, bad things happen https://github.com/forcedotcom/cli/issues/1791
```

Comment:
When I read "bad things happen", my immediate thought is "what bad things". When I read the GH issue, it's not immediately clear what that bad thing is. The only thing I can connect it to is "Command immediately exits with no error", but that doesn't tell me **why**. Can you explain why for the future developer who joins the team in 2 years?

Reply:
I don't understand the **why**